app/init.php

```diff
@@ -3,9 +3,17 @@
 $dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
 $dotenv->load();
 
+$loader = new \Twig\Loader\FilesystemLoader(__DIR__ . '/templates');
+$twigConfig = [];
+if (getenv('PRODUCTION')) {
+    $twigConfig['cache'] = __DIR__ . '/cache';
+}
+$twig = new \Twig\Environment($loader, $twigConfig);
```

Comment:
Finally, this line creates an instance of the Twig template engine (with our config)!